<launch>
 <include file="$(find rplidar_python)/launch/rplidar.launch_v3.xml">
  <arg name="rplidar_frame" value="camera_depth_frame" />
 </include>
 <include file="$(find turtlebot_navigation)/launch/includes/gmapping.launch.xml"/>
 <include file="$(find turtlebot_navigation)/launch/includes/move_base.launch.xml"/>
</launch>

